src/main/resources/lang/strings_en.properties

```diff
@@ -64,6 +64,8 @@ config.general.roaming.keep=Keep roaming towards same point until reached
 config.general.roaming.sequential=Roam to preferred zones in order
 config.general.roaming.only_kill_preferred=Only kill NPCs in preferred area
 config.general.roaming.only_kill_preferred.desc=Only select and kill NPCs when they are inside a preferred area
+config.general.roaming.only_collect_preferred=Only collect resources in preferred area
+config.general.roaming.only_collect_preferred.desc=Only collect resources when they are inside a preferred area
```

Comment:
A description wich is basically identical to the name feels pretty useless, i'd make the name a bit shorter, like:
 `Only collect in prefered area`
and:
`Only search and collect resources when they are inside a prefered area`